velox/functions/prestosql/aggregates/MinMaxAggregates.cpp

```diff
@@ -73,23 +73,53 @@ class MinMaxAggregate : public SimpleNumericAggregate<T, T, T> {
     return sizeof(T);
   }
 
+  int32_t accumulatorAlignmentSize() const override {
+    return 1;
```

Comment:
I suspect if you put 16 bytes as alignment, you can dereference `value<T>` without problem, can you try this?

Reply:
It is 16 bytes aligned:

```
https://github.com/facebookincubator/velox/pull/3755/files#diff-f3a90a5a4a48e4034760e5f6c7ebfd938d3868b0346891cc8fc46e3831adaf6aR118
```